vm_debug.h

```diff
@@ -30,14 +30,17 @@ void ruby_set_debug_option(const char *str);
 
 RUBY_SYMBOL_EXPORT_END
 
+#ifndef RUBY_DEVEL
+#define RUBY_DEVEL 0
+#endif
+
 #if RUBY_DEVEL
 #ifndef USE_RUBY_DEBUG_LOG
-#define USE_RUBY_DEBUG_LOG 0
+#define USE_RUBY_DEBUG_LOG 1
```

Comment:
`USE_RUBY_DEBUG_LOG` introduces a few performance penalty so it should be off by default even if `RUBY_DEVEL` is defined, IMO.

Reply:
OK, I'll make that change.

Reply:
Thank you, no problem on it.